lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -697,14 +697,11 @@ may then be further optimized by selecting better implementations of each operat
                 // A DML operation, such as `INSERT`, `UPDATE` or `DELETE`
                 (dml
                     // The target is an expression that is indicates the table whose data is to be manipulated.
-                    // In practice this can be an identifier or a simplified path expression consisting of only
-                    // literal path steps (and with no wildcard or unpivot operators).  This restriction must
-                    // be enforced by the PartiQL implementation.
-                    // TODO: consider ways to reduce the domain of acceptable values here.  Of course, we it would
-                    // be possible to create a `sum` type like `expr` but containing only the valid DML targets,
-                    // but that would might necessitate duplicating some logic related to variable resolution which is
-                    // already implemented for standard expressions.  This is out of scope for current efforts.
-                    target::expr
+                    // According to the PartiQL's grammar this can be an identifier or a simplified path expression
```

Comment:
By PartiQL grammar do you mean what we have in [type-domain](https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/resources/org/partiql/type-domains/partiql.ion#L376)? If so, can you please clarify? Also maybe it's worthwhile to refer to the RFC PR here as a temporary reference.

Reply:
No, I mean what's supported by `SqlParser` in this project.  I guess, given your DML rfc, we need to start thinking about how your INSERT RFC differs from the reference implementation.

Reply:
That's a fair call, in that case, can we be more explicit in the text b/c that can cause confusion:

```suggestion
                    // With current PartiQL Parser `SqlParser`, this can be an identifier or a simplified path expression
```